examples/custom.py

```diff
@@ -48,10 +47,7 @@ class MyAstroidChecker(BaseChecker):
     def visit_call(self, node: nodes.Call) -> None:
         """Called when a :class:`.nodes.Call` node is visited.
 
-        See :mod:`astroid` for the description of available nodes.
-
-        :param node: The node to check.
```

Comment:
I think this might still be relevant for `sphinx`. We tend to remove `type` in favour of in-line typing, but the `param` lines don't hurt.

Reply:
I thought it does not add any information compared to just the having the name.

Reply:
I don't think we're using it but we do in `astroid`. For example see https://pylint.pycqa.org/projects/astroid/en/latest/api/base_nodes.html#astroid.nodes.NodeNG.scope and how it uses `returns`. I think we should keep these `param` lines in: somebody went through the effort of adding them and we never know when we might want to use it for our own documentation. Doesn't hurt to keep them in.